SmartDeviceLink-iOS/SmartDeviceLink/SDLRPCRequestFactory.h

```diff
@@ -91,6 +94,7 @@
 + (SDLAlert *)buildAlertWithAlertText1:(NSString *)alertText1 alertText2:(NSString *)alertText2 duration:(NSNumber *)duration correlationID:(NSNumber *)correlationID;
 //*****
 
++(SDLAlertManeuver*) buildAlertManeuverwithTTSchunks:(NSMutableArray *)ttsChunks softButtons:(NSMutableArray*) softButtons correlationID:(NSNumber*) correlationID;
```

Comment:
The added methods are improperly formatted. I'm going to reformat everything before release, but please try to stick to proper formatting.